src/sass/Main.scss

```diff
@@ -0,0 +1,12 @@
+// import all sass files here
+// run npm start sass:compile to compile to the public/stylesheet folder
+// this file is imported to App.js
+
+//Abstracts
+@import "Abstracts/Mixins";
+@import "Abstracts/Variables";
+
+//Components
+@import "Components/Home";
+@import "Components/Chat";
```

Comment:
I understand the need to set up boilerplate but the `Chat` and `SideMenu` SCSS files don't yet have the associating component setup yet. Can we leave them out for now until we add those components? I just don't want too much 
 noise and empty files.

Reply:
Yes, that's totally fine, we can remove them